src/py42/clients/orgs.py

```diff
@@ -63,25 +63,24 @@ def get_by_uid(self, org_uid, **kwargs):
         params = dict(idType=u"orgUid", **kwargs)
         return self._session.get(uri, params=params)
 
-    def get_by_name(self, org_name, **kwargs):
+    def get_all_by_name(self, org_name, **kwargs):
         """Gets the organization with the given name.
         `REST Documentation <https://console.us.code42.com/apidocviewer/#Org-get>`__
 
         Args:
             org_name (str): A name of an organization.
 
         Returns:
-            list: A list of :class:`py42.response.Py42Response` objects each containing an
-            organization that has the given name.
+            generator: An object that iterates over :class:`py42.response.Py42Response` objects
+            that each contain a page of organizations with the given name.
         """
         found_orgs = []
         response = self.get_all(**kwargs)
         for page in response:
-            org_list = page[u"orgs"]
-            for org in org_list:
-                if org[u"orgName"] == org_name:
-                    found_orgs.append(org)
-        return found_orgs
+            page[u"orgs"] = [o for o in page[u"orgs"] if o[u"orgName"] == org_name]
+            if page._data_root.get("totalCount"):
```

Comment:
I'm not sure the best way to fix totalCount, since we won't know it from the response. This feels hacky though....

Reply:
Im beginning to feel like this method maybe shouldn't exist since it cant be faithfully represented as `Py42Response`, and that user should just be doing what we're doing here on their own if they need this.

Reply:
Going to delete it